src/components/support-form/SupportForm.tsx

```diff
@@ -148,6 +148,8 @@ const NewsletterDialog = ({ isOpen, handleConfirm, handleCancel }: NewsletterDia
 export default function SupportForm() {
   const { t } = useTranslation()
   const classes = useStyles()
+  const formRef = useRef<FormikProps<SupportFormData>>() as RefObject<FormikProps<SupportFormData>>
```

Comment:
You can avoid setting the second `as` part if you provide a default value of `null` for the ref

```suggestion
  const formRef = useRef<FormikProps<SupportFormData>>(null)
```

This changes the way TS infers the type of the ref from 

```ts
function useRef<T = undefined>(): MutableRefObject<T | undefined>;
```
to
```ts
function useRef<T>(initialValue: T|null): RefObject<T>;
```